main.js

```diff
@@ -189,44 +185,46 @@ class Seq extends utils.Adapter {
 
     onUnload(callback) {
         try {
-            // Flush logger 
+            // Flush logger
             seqLogger.close();
             callback();
         } catch (ex) {
             callback();
         }
     }
 
-    ExtractPidAndMessage(inMessage) {
+    extractPidAndMessage(inMessage) {
         try {
-            const mIndex = Object.values(this.IndexesOf(inMessage, / /g))[0][1];
-            const pIndex = Object.values(this.IndexesOf(inMessage, / /g))[0][0];
+            const mIndex = Object.values(this.indexesOf(inMessage, / /g))[0][1];
+            const pIndex = Object.values(this.indexesOf(inMessage, / /g))[0][0];
             let message = inMessage.substring(mIndex).trim();
             let pid = inMessage.substring(pIndex, mIndex).replace('(', '').replace(')', '').trim();
-    
+
             // check if the message contains a pit, if not the object must fill differently
-            if (isNaN(pid)) {
+            if (isNaN(parseInt(pid, 10))) {
                 message = inMessage.substring(pIndex).trim();
                 pid = -1;
             }
-    
+
             return {
-                Message: message,
-                Pid: parseInt(pid)
+                message,
```

Comment:
Is that really how you just pass the variable without passing it an object variable?

Reply:
I will fix it myself ;)